### PR TITLE
Fix string type name.

### PR DESCRIPTION
**Pull request by trekhleb**
case 1. Add a new type definition.
- [x] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
- [x] has correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)
- [x] has a [test file](http://definitelytyped.org/guides/contributing.html#tests) with the suffix of  `-tests.ts` or `-tests.tsx`.

case 2. Improvement to existing type definition.
- documentation or source code reference which provides context for the suggested changes.  url http://api.jquery.com/html .
  - it has been reviewed by a DefinitelyTyped member.

### CODE REVIEW

File: elastic.js/elastic.js.d.ts [hunk 1]
```diff
@@ -40,7 +40,7 @@ declare module elasticjs {
      Retrieves the internal agg object. This is typically used by
      internal API functions so use with caution.
      */
-    toJSON(): String;
+    toJSON(): Object;
```
Comment (vvakame) [inline elastic.js/elastic.js.d.ts]: you should use `any`.

File: elastic.js/elastic.js.d.ts [hunk 2]
```diff
@@ -139,7 +139,7 @@ declare module elasticjs {
...
```
Comment (vvakame) [inline elastic.js/elastic.js.d.ts]: same above.

File: elastic.js/elastic.js.d.ts (hunk 1 above)
Comment (oleksiii) [inline elastic.js/elastic.js.d.ts]: @vvakame , thank you for review! Type "Object" has been changed to "any". Please review.

File: elastic.js/elastic.js.d.ts (hunk 2 above)
Comment (oleksiii) [inline elastic.js/elastic.js.d.ts]: Yes, all this kind of places has been changed so there are "any" instead of "Object" now. Please check.

**Pull request merged**